validate.sh

```diff
@@ -293,6 +293,15 @@ fi
 # Adjust runtime configuration
 #######################################################################
 
+if [ -z "$JOBS" ]; then
```

Comment:
I'm sorry my bash is very naive, perhaps: if you set $JOBS to empty at the top of this script, can this if-condition ever be false?

Reply:
It can be changed by a `-j` or `--jobs` parameter (check the command line parameters section).

The whole thing is a bit naïve, but that's not introduced by this PR; for example, nothing validates that the jobs parameter is a number.

Reply:
Oh, my bad, thanks!